translations/base-pt-BR.yaml

```diff
@@ -0,0 +1,649 @@
+#
+#  GAME TRANSLATIONS
+#
+#  Contributing:
+#
+#    If you want to contribute, please make a pull request on this respository
+#    and I will have a look.
+#
+#  Placeholders:
+#
+#    Do *not* replace placeholders! Placeholders have a special syntax like
+#    `Hotkey: <key>`. They are encapsulated within angle brackets. The correct
+#    translation for this one in German for example would be: `Taste: <key>` (notice
+#    how the placeholder stayed '<key>' and was not replaced!)
+#
+#  Adding a new language:
+#
+#    If you want to add a new language, ask me in the discord and I will setup
+#    the basic structure so the game also detects it.
+#
+
+global:
+    loading: Carregando
+    error: Erro
+
+    # How big numbers are rendered, e.g. "10,000"
+    thousandsDivider: "."
+    
+    # The suffix for large numbers, e.g. 1.3k, 400.2M, etc.
+    suffix:
+        thousands: K
+        millions: M
+        billions: B
+        trillions: T
+        
+    # Shown for infinitely big numbers
+    infinite: inf
+
+    time:
+        # Used for formatting past time dates
+        oneSecondAgo: um segundo atrás
+        xSecondsAgo: <x> segundos atrás
+        oneMinuteAgo: um minuto atrás
+        xMinutesAgo: <x> minutos atrás
+        oneHourAgo: uma hora atrás
+        xHoursAgo: <x> horas atrás
+        oneDayAgo: um dia atrás
+        xDaysAgo: <x> dias atrás
+
+        # Short formats for times, e.g. '5h 23m'
+        secondsShort: <segundos>s
+        minutesAndSecondsShort: <minutos>m <segundos>s
+        hoursAndMinutesShort: <horas>h <minutos>s
```

Comment:
Quoting from line 9:
```
# Placeholders:
#    Do *not* replace placeholders! Placeholders have a special syntax like
#    `Hotkey: <key>`. They are encapsulated within angle brackets. The correct
#    translation for this one in German for example would be: `Taste: <key>` (notice
#    how the placeholder stayed '<key>' and was not replaced!)```
Do not translate replacements, they are constant strings